src/SourceBuild/content/repo-projects/windowsdesktop.proj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <LogVerbosityOptOut>true</LogVerbosityOptOut>
+
+    <BuildArgs>$(BuildArgs) $(FlagParameterPrefix)v $(LogVerbosity)</BuildArgs>
+    <BuildArgs>$(BuildArgs) $(FlagParameterPrefix)warnAsError $(ArcadeFalseBoolBuildArg)</BuildArgs>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <RepositoryReference Include="arcade" />
+    <!-- https://github.com/dotnet/source-build/issues/3798 -->
+    <RepositoryReference Include="wpf" Condition="'$(TargetOS)' == 'windows'" />
```

Comment:
```suggestion
    <RepositoryReference Include="wpf" />
```

Reply:
?? We don't want to build the wpf repo when not targeting Windows.

Reply:
Ah I see. If we are in the context of windowsdesktop, we already know that TargetOS is windows.